assets/src/blocks/Gallery/GalleryFrontend.js

```diff
@@ -3,34 +3,15 @@ import {GalleryCarousel} from './GalleryCarousel';
 import {GalleryThreeColumns} from './GalleryThreeColumns';
 import {GalleryGrid} from './GalleryGrid';
 import {getGalleryLayout, GALLERY_BLOCK_CLASSES} from './getGalleryLayout';
-import {getCaptionWithCredits} from './getCaptionWithCredits.js';
```

Comment:
Ideally we would also remove the Gallery block from the plugin, since it's moved to the theme already.

Reply:
I agree, but I'd rather do it on another PR.